example/rust-example.md

```diff
@@ -0,0 +1,75 @@
+# Prerequisites
```

Comment:
I prefer to have a folder called `rust-day-1-introduction`. And it should be a complete and seperate project that users can build and run. In the very first example, we will init a fs operator and write/read data from it to show how opendal works in general.

The whole content could be like:

- rust-day-1-introduction
- rust-day-2-init-services
- rust-day-3-xxxx
- python-day-1-introduction
- nodejs-day-1-introduction
- ...